src/pybind/mgr/volumes/fs/operations/versions/subvolume_base.py

```diff
@@ -69,6 +69,19 @@ def base_path(self):
     def config_path(self):
         return os.path.join(self.base_path, b".meta")
 
+    def mark_meta(self):
```

Comment:
Do we still need this with the latest changes? If anything, the doc comment should be updated, as quiescing this file shouldn't be holding the snapshot, or is it still the case?

Reply:
I didn't see any hurt keeping this change in case the root is quiesced. It has no real performance draw-back since we only acquire a rdlock on the xattrlock to first check if it's already set (same as the subvolume flag).

However, for keeping the scope of changes smaller, I will revert that commit in the series. (but keep it in the history)

Reply:
I'm keeping it in the history so there is a direction shift in the code after switching to the local lock. In that case, we do not need to block quiesce on the meta file.